index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <span class="my-logo"><a href="#">My Logo</a></span>
+        <nav>
+          <a href=""><img src="assets/images/Union.png" alt="Union-icon" /></a>
+        </nav>
+      </header>
+
+      <section class="hero">
+        <div class="greeting">
+          <p>
+            I'm Zerihun. <br />
+            Glad to see <br />
+            you!
+          </p>
+        </div>
+        <div class="job-description">
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
+        <span class="connect-link">LET'S CONNECT</span>
+        <div class="social-media-icons">
+          <ul class="social-media-links">
+            <li>
+              <a href="#"
+                ><img src="assets/images/twitter-logo.png" alt="twitter-logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="assets/images/Linkedin icon.png" alt="linkedin-logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="assets/images/medium-icon.png" alt="medium-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/images/github-logo.png" alt="github-logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="assets/images/angel-list-logo.png"
+                  alt="angel-list-logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
```

Comment:
- [ ] kindly consider adding the background image properly as given in the provided design to deliver the expected output check the images below to see the difference you made 
- your design 👇 
<img width="369" alt="Screen Shot 2023-04-28 at 15 05 34" src="https://user-images.githubusercontent.com/60146030/235155724-ee82ae1a-d32e-4078-a557-9ecda16f2643.png">

- figma design 👇 
<img width="383" alt="Screen Shot 2023-04-28 at 15 06 28" src="https://user-images.githubusercontent.com/60146030/235155837-1e6bc6f3-2c39-445b-aada-1ae056d3b06c.png">